statsmodels/base/_penalized.py

```diff
@@ -54,13 +54,33 @@ def __init__(self, *args, **kwds):
         self._null_drop_keys = getattr(self, '_null_drop_keys', [])
         self._null_drop_keys.extend(['penal'])
 
+    def _handle_scale(self, params, **kwds):
+
+        #if kwds['scale'] is not None:
+        #    raise
```

Comment:
raise something in particular?  why not just put `scale=None` in the function definition?

Reply:
plain `raise` are most likely leftovers from debugging.
It's a simple way to create a break point for `pytest --pdb`, and sometimes I use it in an if-else to make sure I'm on the correct branch.

Reply:
I changed it to `scale=None` keyword argument